source/php/Modules/FrontendForm/FrontendForm.php

```diff
@@ -19,18 +19,23 @@ class FrontendForm extends \Modularity\Module
     public $slug = 'event-form';
     public $supports = [];
 
+    // The field groups that should be displayed in the form.
+    private $fieldGroups = [
+      'group_65a115157a046'
```

Comment:
Can we make this less coupled directly to the acf settings? Through passing constructor params maybe?